opta-mern-stack/README.md

```diff
@@ -0,0 +1,114 @@
+![alt text](MERN.jpb "MERN Stack MongoDB, Express, React, Node.js")
+## Introduction
+
+MERN stack is a web software stack that includes MongoDB, Express.js, React, and Node.js. In this post we show how you can use [Opta](https://docs.opta.dev/) to easily stand up a MERN-based application stack on a public cloud (gcp/aws) or Local development Kubernetes cluster. Specifically:
+
+1. An [AWS eks/Google cloud gke/Local PC Kind] Kubernetes cluster built to SOC2 security standards
+
+2. A React single page app, containerized to run in your Kubernetes cluster
+
+3. A Nodejs + Express app, containerized to run in your Kubernetes cluster
+
+4. A MongoDB Atlas cluster to serve as the persistent database layer
+
+
+## Pre-requisites
+
+Opta is a new kind of Infrastructure-As-Code framework where you work with high-level constructs instead of getting lost in low level cloud configuration or having to deal with [Terraform HCL](https://blog.runx.dev/my-pet-peeves-with-terraform-f9bb37d94950). You can install Opta on your PC by following the instructions [here](https://docs.opta.dev/installation/).
+
+Our MERN stack example is based on the application code created by The MongoDB Atlas team. Their [blog post](https://www.mongodb.com/languages/mern-stack-tutorial) will walk you through the Nodejs+Express backend, React frontend and MongoDB code. You should take a moment to read the blog post, although keep in mind that most of the manual/click-on-GUI steps mentioned in the post to setup the infrastructure are not applicable to this Opta Infrastructure-as-code MERN stack example.
+
+You can choose to deploy the MERN stack application on Kubernetes running in AWS (EKS cluster), GCP (GKE cluster) or Locally on your laptop (Kind cluster). AWS and GCP will incur cloud charges, but the local setup is completely free. We also use a MongoDB Atlas database, we will sign up for their service and use their a small database (could cost upto $3 per day); you can always scale up the MongoDB deployment for your production code later.
```

Comment:
add a link to the `kind`. Many people might not know what that is

Reply:
Done